### PR TITLE
Fix i2c parameter documentation

### PR DESCRIPTION
**Pull request by spovlot**

### CODE REVIEW

**Comment by tekktrik** [comment]
Just wanted to say that you can ignore the warning about the exceptions, I'll fix that this week.  The "line too long" one should get fixed however.

File: adafruit_servokit.py [hunk 1]
```diff
@@ -61,8 +61,7 @@ class ServoKit:
 
     :param int channels: The number of servo channels available. Must be 8 or 16. The FeatherWing
                          has 8 channels. The Shield, HAT, and Bonnet have 16 channels.
-    :param ~I2C i2c: The I2C bus to use. If not provided, it will use generate the default I2C
-                                      bus singleton ``busio.I2C()`` and use that.
+    :param ~I2C i2c: The I2C bus to use. If not provided, the default I2C ``board.I2C()`` will be used.
```
Comment (tekktrik) [inline adafruit_servokit.py:64]: Looks like this just needs to be split into two lines like some of the other docstring params here.

**Review by tekktrik — CHANGES_REQUESTED** [review]

**Comment by tekktrik** [comment]
Looks like running `pre-commit` (specifically `black`) one last time should fix it now!

**Review by FoamyGuy — APPROVED** [review]
Looks good. Thank you @spovlot!

**Pull request merged**